packages/swingset-theme-hashicorp/src/components/side-nav/link-item.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "../link"
+import { cx } from "class-variance-authority"
+
+function LinkItem({ title, to }: Record<'title' | 'to', string>) {
```

Comment:
The use of `Record` is clever here, but I think it'd be clearer if we used a more conventional type definition:
```suggestion
type LinkItemProps = {
	title: string
	to: string
}

function LinkItem({ title, to }: LinkItemProps) {
```